libraries/AP_Notify/NotifyDevice.h

```diff
@@ -14,6 +16,9 @@ class NotifyDevice {
     virtual void update() = 0;
     // handle a LED_CONTROL message, by default device ignore message
     virtual void handle_led_control(mavlink_message_t *msg) {}
+
+    // this pointer is used to read the parameters relative to devices
+    AP_Notify *pNotify;
```

Comment:
Can we make this a reference please?  A pointer can be NULL whereas a reference never can be so we have a preference for that.  Thanks.